From 7d1a57b1ecd4f1e82d4123afb9c7c22078811816 Mon Sep 17 00:00:00 2001
From: Pierre-Jean TEXIER <pjtexier@amplitude-systemes.com>
Date: Wed, 14 Sep 2016 10:56:50 +0200
Subject: [PATCH] change Board name

Signed-off-by: Pierre-Jean TEXIER <pjtexier@amplitude-systemes.com>
---
 board/warp7/warp7.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/warp7/warp7.c b/board/warp7/warp7.c
index 27e31f3..d4774be 100644
--- a/board/warp7/warp7.c
+++ b/board/warp7/warp7.c
@@ -95,7 +95,7 @@ int board_init(void)
 
 int checkboard(void)
 {
-	puts("Board: WARP7\n");
+	puts("Board: WARP7 - bdx iot\n");
 
 	return 0;
 }
-- 
1.9.1

